crates/sui-framework/packages/deepbook/sources/math.move

```diff
@@ -20,11 +29,71 @@ module deepbook::math {
         (is_round_down, ((x * y / FLOAT_SCALING_U128) as u64))
     }
 
+    public fun mul_round_check_underflow(x: u64, y: u64): (bool, u64) {
```

Comment:
For each usage of `clob_math::mul` or `clob_math::div_round` in clob.move, could you please add a detailed comment explaining why underflow is acceptable?

Reply:
We add comments for why underflow is permitted in clob.move.

Reply:
Upon further reflection, I was wondering if we shouldn't switch these--e.g., `mul` is the function that checks for overflow, and the one that does not is called `unsafe_mul` or `mul_with_underflow_risk` or similar so the caller is aware of the risk. That feels safer than a comment--WDYT?